apps/web/src/constants/routes.ts

```diff
@@ -60,4 +60,4 @@ export enum ROUTES {
   STUDIO_FLOWS_TEST_STEP = '/studio/flows/:templateId/step/:stepId/test',
 }
 
-export const PUBLIC_ROUTES_PREFIXES = new Set<string>(['/auth', '/partner-integrations']);
+export const PUBLIC_ROUTES_PREFIXES = new Set<string>(['/auth']);
```

Comment:
Removed `partner-integrations` from public routes because as of now we only have vercel integration and it needs an active session